README.md

```diff
@@ -478,6 +478,9 @@ Or as a param in `synchronize` method:
 Photo.synchronize(transaction_per_page: true)
 ```
 
+## Model Aliasing
+By default `synced` gem will look for an endpoint in BookingSync with the same name as it was defined in. If you want to name your class differently or place it in a namespace, provide the actual resource name with the `endpoint` option. like so: `synced endpoint: "Booking"`.
```

Comment:
It's not entirely true, it will look for pluralized name. I'd rather go with `provide endpoint of resource that you want to access` or sth similar. With current code this example `synced endpoint: "Booking"` will fail - `synced endpoint: "bookings"` would be the correct one (as you did in spec). Also `like so` with capital letter.